example/python/tx-example.py

```diff
@@ -188,11 +210,26 @@ def get_account_asset():
 
     print(query_response)
 
+def get_user_detail():
+    # get detail of the user
+    query = query_builder.creatorAccountId(creator) \
+        .createdTime(current_time) \
+        .queryCounter(11) \
```

Comment:
Why is it 11?

Reply:
If you want to get unique query counter, maybe use a static variable with increment instead?